androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TrustedWebActivityService.java

```diff
@@ -14,11 +14,18 @@
 
 package com.google.androidbrowserhelper.trusted;
 
-import androidx.browser.trusted.TrustedWebActivityService;
+import androidx.annotation.NonNull;
+import androidx.browser.trusted.TokenStore;
 
 /**
- * An extension of {@link TrustedWebActivityService} to allow us to extend its functionality in the
- * future without requiring clients to change their code.
+ * An extension of {@link androidx.browser.trusted.TrustedWebActivityService} to allow us to extend
+ * its functionality in the future without requiring clients to change their code.
```

Comment:
This comment is no longer accurate - this class now provides implementation for the TrustedWebActivityService.

I remember creating this class early on so that we could make this change in a non breaking way. I had hoped that people would include the `DelegationService` in their manifest instead of AndroidX's `TrustedWebActivityService`. So I think that if we just left the name as-is, this wouldn't be a breaking change (or at least the breaking change would be in AndroidX, making TrustedWebActivityService abstract, not in the Android Browser Helper Library).

Can you remind me which part of the TrustedWebActivityService you think didn't fit the name DelegationService?

Reply:
We can keep it as `DelegationService`, but the break will come from replacing the `androidx.browser.trusted.TrustedWebActivityService` with `com.google.androidbrowserhelper.trusted.TrustedWebActivityService`. svgomg-twa and llama-pack use TrustedWebActivityService. :(

Reply:
done